src/modules/configurator/components/ConfigPropertyView.tsx

```diff
@@ -0,0 +1,31 @@
+import { ConfigProperty } from "./ConfigProperty";
+import { CarProperty } from "../types";
+
+export function ConfigPropertyView({
+  index,
+  propertyName,
+  modelName,
+  label,
+  name,
+  description,
+  price,
+}: CarProperty) {
+  return (
+    <div className="flex items-center justify-between cursor-text">
+      <ConfigProperty
+        index={index}
+        propertyName={propertyName}
+        modelName={modelName}
+        label={label}
+        name={name}
+        description={description}
+        price={price}
+        isDescriptionDisplayed={false}
```

Comment:
You dont need to pass the false value if this is optional. Your description wont be visible if this value is undefined.

Reply:
Removed ` isDescriptionDisplayed={false}`.